tests/test_metrics.py

```diff
@@ -14,6 +14,7 @@
     LogNormalDistributionLoss,
     NegativeBinomialDistributionLoss,
     NormalDistributionLoss,
+    BetaDistributionLoss,
```

Comment:
Could you run black and isort (best is to use the pre-commit hooks to ensure correct formatting)?